src/main/java/com/codecool/dungeoncrawl/logic/actors/Key.java

```diff
@@ -0,0 +1,38 @@
+package com.codecool.dungeoncrawl.logic.actors;
+
+import com.codecool.dungeoncrawl.logic.Cell;
+
+public class Key {
```

Comment:
Polymorphism could be used to skip code duplication: what about an abstract class above Key and Sword